### PR TITLE
Update amplifysetup.md

### PR DESCRIPTION
**Pull request by saka015**

### CODE REVIEW

File: development/setup/amplifysetup.md [hunk 1]
```diff
@@ -1 +1,104 @@
...
+## setting up amplify
+### run these commands
+
+amplify configure
+press ENTER (x2)
+
+Enter the access key of the newly created user: 
+
+accessKeyId: (put_your_access_ID)
+ secretAccessKey:  (put_your_access_KEY)
```
Comment (nbnw-org) [inline development/setup/amplifysetup.md]: plese fix formatting (extra space)

File: development/setup/amplifysetup.md [hunk 2]
```diff
@@ -1 +1,104 @@
...
+(terminal will get exit() , run below commands)
+ amplify pull
+
+ Select the authentication method you want to use: AWS profile
+ Please choose the profile you want to use default  
+ Which app are you working on? (choose_your_app_name)
+ Choose your default editor: Visual Studio Code     
+Choose the type of app that you're building · javascript
+
+### Please tell us about your project
```
Comment (nbnw-org) [inline development/setup/amplifysetup.md]: can you add a note at start as well telling that there are changes in Distribution Directory path, do take note of that... Generally people have a habbit of skimming through...

File: development/setup/amplifysetup.md [hunk 3]
```diff
@@ -1 +1,104 @@
 # Owner - Kamal
+
+# Setting Up an Amplify Full-Stack App
+This guide walks you through creating a full-stack application using Amplify. We'll cover the essential steps to get your project up and running.
```
Comment (nbnw-org) [inline development/setup/amplifysetup.md]: Can we separate it into following sections - 
1. Setting up CLI (which includes the access key part)
2. Setting up a new project
3. Setting up an existing project

And update the details accordingly?

**Comment by nbnw-org** [comment]
@saka015 , please add revision to this... Thanks!

File: development/setup/amplifysetup.md (hunk 3 above)
Comment (saka015) [reply development/setup/amplifysetup.md]: okay will update it

**Comment by nbnw-org** [comment]
@saka015 what is the status of the revision? Thanks!

**Pull request merged**